code_generation/analysis_template.cxx

```diff
@@ -48,9 +48,22 @@ int main(int argc, char *argv[]) {
         Logger::get("main")->info("Running with {} input files", argc - 2);
     }
     std::vector<std::string> input_files;
+    int nevents = 0;
     for (int i = 2; i < argc; i++) {
```

Comment:
Why does it start with 2? Does the argv counter start with 1? I thought the the first input file is at position >=1 and the output file at position 0.
Also [here](https://github.com/KIT-CMS/CROWN/blob/77d02ce24e8af554c4665f8d470abbbac5388f84/code_generation/analysis_template.cxx#L47) shouldn't this be >=3 and not >3?

Reply:
Then never mind, thanks for clarifying. But the second comment with >=3 is still valid, right? You could have only one input file, then argc would be 3 and this case is not covered.

Reply:
I will just remove the printout